easybuild/easyconfigs/c/Cython/Cython-0.29.34-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'PythonPackage'
+
+name = 'Cython'
+version = '0.29.34'
```

Comment:
Do we really need this? Often i suspect people updating this easyconfig because we had some from the past, but Python bundle itself includes Cython (though a bit older, from when the 3.9.5 Python bundle was created)

Reply:
thanks, I will remove Cython (it was inherited from the 2020a toolchain)

Reply:
done